### DISCUSSION TITLE
New install of TeamPass 3.10  issue

### DISCUSSION THREAD

**Discussion by Aviad48** [nilsteampassnet/TeamPass#3969] [General]
hello,
i am installing a fresh install of TeamPass 3.10  using docker compose i get to the start menu and when i start i get the following error as you can see in the pic attach as well
"File includes/config/settings.php could not be copied from includes/config/settings.sample.php.
Please do it on your own or change folder rights, and click START button!"

will be happy for help on this.

![tp](https://github.com/nilsteampassnet/TeamPass/assets/20218671/0d30bf67-bf86-459b-9210-9f50ebabde48)

**Comment by nilsteampassnet** [comment]
@Aviad48 
Can you please chmod the folder `includes/config` to permit Apache to write in it.

**Comment by Aviad48** [comment]
I manage to fix it  with a  older version 3.9
It looks like a bug in 3.10

**Comment by wander0987** [comment]
Hi, how do you install an older version?
image: dormancygrace/teampass:latest is not working for me and i get an error when I try image:
dormancygrace/teampass:3.1.0
or 
dormancygrace/teampass:3.0.9

Best regards
